src/src/com/microsoft/aad/adal/BrokerProxy.java

```diff
@@ -447,6 +454,26 @@ private Bundle getBrokerOptions(final AuthenticationRequest request) {
         }
         return brokerOptions;
     }
+    
+    /**
+     * Check if the broker is the new one with PRT support by checking the version returned from intent. 
+     * Only new broker will send {@link AuthenticationConstants.Broker.BROKER_VERSION}, and the version number
+     * will be v2. 
+     */
+    private boolean isBrokerWithPRTSupport(final Intent intent) {
+        if (intent == null) {
+            throw new IllegalArgumentException("intent");
+        }
+        
+        // Only new broker with PRT support will send down the value and the version will be v2
+        final String brokerVersion = intent.getStringExtra(AuthenticationConstants.Broker.BROKER_VERSION);
+        if (!StringExtensions.IsNullOrBlank(brokerVersion) 
+                && AuthenticationConstants.Broker.BROKER_PROTOCOL_VERSION.equalsIgnoreCase(brokerVersion)) {
```

Comment:
you can do 
return (!StringExtensions.IsNullOrBlank(brokerVersion) 
                && AuthenticationConstants.Broker.BROKER_PROTOCOL_VERSION.equalsIgnoreCase(brokerVersion));